core/modules/url/helpers/index.ts

```diff
@@ -1,64 +1,55 @@
 import { router } from '@vue-storefront/core/app'
 import config from 'config'
-import { localizedDispatcherRoute, localizedRoute, LocalizedRoute } from '@vue-storefront/core/lib/multistore'
+import { localizedDispatcherRoute, localizedRoute, LocalizedRoute, currentStoreView } from '@vue-storefront/core/lib/multistore'
 import { RouteConfig } from 'vue-router/types/router';
 import { RouterManager } from '@vue-storefront/core/lib/router-manager'
-import { currentStoreView } from '@vue-storefront/core/lib/multistore'
 import { Category } from 'core/modules/catalog-next/types/Category'
 import { removeStoreCodeFromRoute } from '@vue-storefront/core/lib/multistore'
+import { Logger } from '@vue-storefront/core/lib/logger'
 
 export function parametrizeRouteData (routeData: LocalizedRoute, query: { [id: string]: any } | string, storeCodeInPath: string): LocalizedRoute {
   const parametrizedRoute = Object.assign({}, routeData)
   parametrizedRoute.params = Object.assign({}, parametrizedRoute.params || {}, query)
-  if (storeCodeInPath && !parametrizedRoute.name.startsWith(storeCodeInPath + '-')) {
-    parametrizedRoute.name = storeCodeInPath + '-' + parametrizedRoute.name
-  }
+  // TODO - need route name prefix? (if not storeCodeInPath not needed)
+  // if (storeCodeInPath && !parametrizedRoute.name.startsWith(storeCodeInPath + '-')) {
+  //   parametrizedRoute.name = storeCodeInPath + '-' + parametrizedRoute.name
+  // }
   return parametrizedRoute
```

Comment:
I think we don't need it however it should be tested if there won't be duplicates